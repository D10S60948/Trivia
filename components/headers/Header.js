import React, { Component } from 'react'
import { StyleSheet, View, Text, Image } from 'react-native'

export default class Header extends Component {

    render() {
        return (
            <View style={styles.container}>    
                <Image source={require('../../images/logo4.png')} style={styles.image} resizeMode='stretch' />
                <View style={styles.container}>
                    <Text style={{...styles.text, fontSize: 50, marginBottom: -30}}>COUNTRIES</Text>
                    <Text style={{...styles.text, fontSize: 70}}>TRIVIA</Text>
                </View>
            </View>      
        );
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 2,
        alignItems: 'center',
        justifyContent: 'center',
        flexDirection: 'column'
    },
    text: { 
      fontFamily: 'Modak',
      textShadowOffset: {width: 3,height:2},
      color:'#1F1F3F'
    },
    image: {
        position: 'absolute',
        left:-10,
        top:110,
        width:120, 
        height:120
    }
});