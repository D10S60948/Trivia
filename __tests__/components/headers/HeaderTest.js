import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header  from '../../../components/headers/Header';

test('"Header" component renders correctly', () => {
    const snapshot = renderer.create(<Header/>).toJSON();
    expect(snapshot).toMatchSnapshot();
});
