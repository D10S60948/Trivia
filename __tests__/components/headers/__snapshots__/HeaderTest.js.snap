// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"Header" component renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 2,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <Image
    resizeMode="stretch"
    source={
      Object {
        "testUri": "../../../images/logo4.png",
      }
    }
    style={
      Object {
        "height": 120,
        "left": -10,
        "position": "absolute",
        "top": 145,
        "width": 120,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 2,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#1F1F3F",
          "fontFamily": "Modak",
          "fontSize": 50,
          "marginBottom": -30,
          "textShadowOffset": Object {
            "height": 2,
            "width": 3,
          },
        }
      }
    >
      COUNTRIES
    </Text>
    <Text
      style={
        Object {
          "color": "#1F1F3F",
          "fontFamily": "Modak",
          "fontSize": 70,
          "textShadowOffset": Object {
            "height": 2,
            "width": 3,
          },
        }
      }
    >
      TRIVIA
    </Text>
  </View>
</View>
`;
